docs/typescript/testing.md

```diff
@@ -2,92 +2,246 @@
 id: testing
 title: Testing TypeScript Workflows
 sidebar_label: Testing
-description: The TypeScript SDK does not yet have a test suite that allows time skipping.
 ---
 
-import CustomWarning from "../components/CustomWarning.js"
+The TypeScript SDK comes with an optional test framework (npm [`@temporalio/testing`](https://www.npmjs.com/package/@temporalio/testing)).
```

Comment:
```suggestion
The TypeScript SDK comes with an optional test framework (`@temporalio/testing`).
```

Purpose in linking to npm? I'll add README, but it should just point back to this page.

Reply:
Yes, why not?

Reply:
I added a README already

Reply:
Seems inefficient to direct reader's attention away ("oh there's a test framework, I'll go read about it. wait there's nothing here, okay I'll go back") for no purpose

Reply:
Good point, I'll submit a new PR to address